cpp/ql/src/Security/CWE/CWE-311/CleartextTransmission.ql

```diff
@@ -168,6 +168,14 @@ class NetworkRecv extends NetworkSendRecv {
   override Recv target;
 }
 
+predicate encryptionFunction(Function f) {
+  f.getName().toLowerCase().regexpMatch(".*(crypt|encode|decode|hash|securezero).*")
+}
+
+predicate encryptionType(UserType t) {
+  t.getName().toLowerCase().regexpMatch(".*(crypt|encode|decode|hash|securezero).*")
+}
```

Comment:
If this is for a join-order fix you probably want to mark them as `pragma[noinline]`.

Reply:
Done (and improves performance a bit further)!

Reply:
Perfect! Do you happen to have tuple counts before and after this change?